apis/server/system_bridge.go

```diff
@@ -123,11 +121,17 @@ func (s *Server) events(ctx context.Context, rw http.ResponseWriter, req *http.R
 		return nil
 	}
 
+	// Notes(ziren): The pouchd http server set some http timeout parameters (see pouch/apis/server/server.go),
+	// that may cause long idle connections been recycled. So, creating a tick here to keep the
+	// event api alive. set the ticker to 8s because of the IdleTimeout of http server is 10s.
+	tickChan := time.NewTicker(time.Second * 10).C
```

Comment:
I think we should use `time.NewTicker` and `defer tick.Stop()` here, because we don't want to leave it active in the timer heap.

Reply:
I am afraid we may not need the `defer` because i checked the example of `NewTicker` when we should add `defer tick.Stop()` when using `NewTicker`, but not need the `Stop` when using `NewTicker().C`

Reply:
OK, when i read the test cases in `time` package, i realize i am wrong. we should call `defer tick.Stop()` when creat a ticker.